src/project/mod.rs

```diff
@@ -226,6 +248,29 @@ impl Project {
         })
     }
 
+    /// Add a platform to the project
+    pub fn add_platforms<'a>(
+        &mut self,
+        platforms: impl Iterator<Item = &'a Platform> + Clone,
+    ) -> miette::Result<()> {
+        // Add to platform table
+        let platform_array = &mut self.doc["project"]["platforms"];
+        let platform_array = platform_array.as_array_mut().ok_or_else(|| {
```

Comment:
I think you can just unwrap this since otherwise parsing of the project would also have failed.